apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -217,15 +224,18 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
         addUnhashedIdentifier(
             patient,
             mbi,
-            TransformerConstants.CODING_BBAPI_MEDICARE_BENEFICIARY_ID_UNHASHED,
+            TransformerConstants.CODING_SYSTEM_HL7_IDENTIFIER_TYPE,
             historicalIdentifier);
       }
     }
 
+    patient.setActive(!beneficiary.getBeneficiaryDateOfDeath().isPresent());
```

Comment:
`active` is defined as:
> Whether this patient's record is in active use

Based on the IG, I would set `deceasedBoolean` to `false` if `getBeneficiaryDateOfDeath()` isn't present.  On like 264 below they set `deceasedDateTime` if it _is_ present.  Only one of those two fields needs to be set though (i.e.: don't set `deceasedBoolean: false` when you set `deceasedDateTime`)